api.js

```diff
@@ -1088,20 +1285,97 @@ module.exports = async function attachAPI(app, {wss, db}) {
 
       const serializedUser = await serialize.user(user, user)
 
+      let authorizationMessage, userAuthorized
+      if (await shouldUseAuthorization()) {
+        userAuthorized = await isUserAuthorized(user._id)
+        if (!userAuthorized) {
+          authorizationMessage = (
+            await db.settings.findOne({_id: serverSettingsID})
+          ).authorizationMessage
+        }
+      }
+
       response.status(200).end(JSON.stringify({
         success: true,
         user: serializedUser,
+
+        // This is redundant since it's already stored on the user, but it's
+        // nice to have anyways - it makes the "authorization message" property
+        // seem less out of place.
+        userAuthorized,
+        authorizationMessage
```

Comment:
Yes, why _is_ `authorizationMessage` given at this endpoint? Surely it should be given at `should-use-authorization`?

Reply:
I don't think should-use-authorization existed when this was implemented - good idea!